invokeai/app/services/model_install/model_install_default.py

```diff
@@ -660,13 +661,96 @@ def _probe(self, model_path: Path, config: Optional[ModelRecordChanges] = None):
+        image_extensions = ['.png', '.jpg', '.jpeg', '.webp']
+        preview_image_path = None
+        
+        for ext in image_extensions:
+            image_path = model_dir / f"{model_stem}{ext}"
+            if image_path.exists():
+                preview_image_path = image_path
+                self._logger.info(f"Found preview image {image_path.name} for LoRA model {model_path.name}")
+                break
+        
+        # Set the preview image if found
+        if preview_image_path:
+            # Store the relative path to the image file
+            if preview_image_path.is_relative_to(self.app_config.models_path):
+                relative_path = preview_image_path.relative_to(self.app_config.models_path)
+                info.cover_image = relative_path.as_posix()
+            else:
+                info.cover_image = preview_image_path.as_posix()
+            self._logger.info(f"Set cover_image to {info.cover_image} for LoRA model {model_path.name}")
+        
+        # Look for JSON file with same name
+        json_path = model_dir / f"{model_stem}.json"
+        
+        if json_path.exists():
+            try:
+                with open(json_path, 'r', encoding='utf-8') as f:
+                    metadata = json.load(f)
+                
+                # Check if the JSON has any of the expected keys
+                expected_keys = {
+                    "description", "sd version", "activation text", 
+                    "preferred weight", "negative text", "notes"
+                }
+                
+                if any(key in metadata for key in expected_keys):
+                    # Map description + notes to model description
+                    description_parts = []
+                    if "description" in metadata and metadata["description"]:
+                        description_parts.append(str(metadata["description"]).strip())
+                    if "notes" in metadata and metadata["notes"]:
+                        description_parts.append(str(metadata["notes"]).strip())
+                    
+                    if description_parts:
+                        combined_description = " | ".join(description_parts)
+                        info.description = combined_description
+                    
+                    # Map activation text to trigger phrases
+                    if "activation text" in metadata and metadata["activation text"]:
+                        activation_text = str(metadata["activation text"]).strip()
+                        if activation_text:
+                            # Split on commas and clean up each phrase
+                            phrases = [phrase.strip() for phrase in activation_text.split(',')]
+                            phrases = [phrase for phrase in phrases if phrase]  # Remove empty strings
+                            if phrases:
+                                info.trigger_phrases = set(phrases)
```

Comment:
This section feels overly complicated. 

```suggestion
                # Map description + notes to model description
                description_parts: list[str] = []
                if description := metadata.get("description", None):
                    description_parts.append(description.strip())
                if notes := metadata.get("notes", None):
                    description_parts.append(notes.strip())

                if description_parts:
                    combined_description = " | ".join(description_parts)
                    info.description = combined_description

                # Map activation text to trigger phrases
                if activation_text:= metadata.get("activation text", None):
                    # Split on commas and clean up each phrase
                    phrases = [phrase.strip() for phrase in activation_text.split(',')]
                    phrases = [phrase for phrase in phrases if phrase]  # Remove empty strings
                    if phrases:
                        info.trigger_phrases = set(phrases)
```